00_hello_world/main.go

```diff
@@ -9,5 +9,21 @@ import (
 var out io.Writer = os.Stdout
 
 func main() {
-	fmt.Fprintln(out, "Hallo du schöne Welt!")
+	fib(7)
+}
+
+func fib(n uint) {
+	if n == 0 {
+		return
+	}
+	f1 := 0
+	f2 := 1
+	fmt.Fprintln(out, f2)
```

Comment:
It's better to decouple fibonacci sequence calculation from IO. https://github.com/anz-bank/go-training/issues/37

Reply:
Agree with you but in Lab it was asked to print in the function. Will keep this mind in future labs.